fern/pages/welcome.mdx

```diff
@@ -15,31 +15,30 @@ layout: overview
   >
     Start with an API Definition
   </Card>
-  <Card
-    title="Docs"
-    icon="fa-solid fa-browser"
-    href="/learn/docs/getting-started/quickstart"
-  >
-    A beautiful documentation website
-  </Card>
   <Card
     title="SDKs"
     icon="fa-brands fa-codepen"
     href="/learn/sdks/introduction/overview"
   >
     Idiomatic client libraries
   </Card>
+  <Card
+    title="Docs"
+    icon="fa-solid fa-browser"
+    href="/learn/docs/getting-started/quickstart"
+  >
+    A beautiful documentation website
+  </Card>
 </Cards>
 
 ## Motivation
 
 <div className="motivation">
   Stripe, Twilio, and AWS have the resources to invest in internal tooling for developer 
-  experience. They provide client libraries in multiple languages and developer documentation
-  that stays up-to-date. 
+  experience. They provide client libraries in multiple languages and up-to-date developer documentation. 
 
-  We are building Fern to productize this process and make it accessible to all
-  software companies.
+  We built Fern to productize this process and make it accessible. Just input an OpenAPI
```

Comment:
APIs underpin the internet but are still painful to work with. They are often untyped, unstandardized, and out-of-sync across multiple sources of truth.

While companies like Stripe, Twilio, and AWS have the resources to develop internal tools for developer experience. Not all businesses have this advantage.

That's why we are building Fern. Fern allows engineering teams to offer developer-friendly APIs, including easy-to-use SDKs and up-to-date developer documentation.